src/Symfony/Component/Security/Http/Event/TokenDeauthenticatedEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Event;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ */
```

Comment:
Probably still a TODO for you, but this needs a nice description of its purpose, especially as it's very similar in name to `DeauthenticatedEvent` - there may be some confusion.

Reply:
Oh definitely, haven't looked at a single PHPdoc yet.